nodes/mainnet.md

```diff
@@ -234,9 +233,19 @@ to see health/uptime status of DA bootstrappers (now `celestia` network only).
 
 You can [find the status of these endpoints](https://celestia-tools.brightlystake.com/).
 
+#### Archival DA RPC endpoints
+
```

Comment:
- [ ] add context on what these type of endpoints are before jumping straight to Grove specifics

Reply:
Done.